keyboard.html

```diff
@@ -64,16 +68,17 @@
       document.getElementById("description").innerHTML = `Below should be a keyboard with ${totalKeyCount} keys.`;
     }
 
-    function addBlackKey() {
+    function addBlackKey(keyCount) {
       const newDiv = document.createElement("div");
       newDiv.classList.add('black-key')
+      newDiv.setAttribute('key', `${keyNames[keyCount]}`)
       keyboard.appendChild(newDiv);
       sizeBlackKey(newDiv)
     }
 
-    function blackKeyCheck(whiteKeyCount) {
-      const whiteKeysNeedingBlackKeys = [0,1,3,4,5]
-      return whiteKeysNeedingBlackKeys.includes(whiteKeyCount)
+    function blackKeyCheck(keyCount) {
+      const whiteKeysNeedingBlackKeys = [0,1,2,5,7,9]
```

Comment:
This is telling me that the `keyNames` object ought to be expanded. off the top of my head I could imagine it as such:
```javascript
keys = {
 0: { "needsBlackKey": true, "name": "C" },
 ...
}
```

Reply:
i imagine i might need to do something along these lines down the road yeah...

Reply:
great callout thanks DJ!